controllers/mocks/HydraClientInterface.go

```diff
@@ -0,0 +1,101 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
Please add section development in README with information about mockery version and how to re-generate mocks